Лаб 1/1.py

```diff
@@ -0,0 +1,19 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+
+numbers1 = numbers[5:] # после неизвестного
+numbers2 = numbers[:4] # до неизвестного
+n = numbers2 + numbers1
+
+l = len(n)
```

Comment:
Вот этой строке допущена ошибка, так как по условию задачи, среднее арифметическое берется от всего списка `numbers`, собственно нам и необходимо узнать количество элементов в исходном списке. При этом наличие в списке не только чисел - не помеха для этого метода `len()` он только смотрит сколько значений, а какие они - ему не важно.
```suggestion
list_lenght = len(numbers)
```
Не забывайте - именовать переменные надо четко и понятно, чтобы другой человек понимал какими переменными вы оперируете и какие функции с чем работают